### ISSUE TITLE
Unable to Activate Power Toggle without Entering Edit Mode

### ISSUE THREAD

**Issue by Xatinni** [turing-machines/BMC-UI#22]
In the new 2.5 TPi platform, the power toggle button appears but is inactive until entering "Edit Mode" by pressing the edit button. This design choice isn't immediately clear to users and can cause confusion, as it suggests the toggle is directly actionable without needing additional steps.

**Steps to Reproduce:**
1. Open the BMC-UI on the 2.5 TPi - BMC UI v3.3.3.
2. Observe the power toggle button, which is visible but inactive.
3. Attempt to activate the toggle without pressing the edit button – it does not respond.

**Expected Behavior**: 
When the power toggle button is visible, it should either be actionable without needing to enter edit mode, or the UI should indicate that edit mode must be activated first.

**Suggested Enhancements:**
Add a tooltip or message indicating that the edit button needs to be pressed to enable the power toggle.
Consider allowing the power toggle to be directly active or visually showing it as disabled with a prompt to enter edit mode.

**Comment by barrenechea** [comment]
I agree on that it isn't the most fluid experience for the end-user... I opened a proposal for this on #26, let me know what you think!
